### PR TITLE
DMU + DMC: Temporal Firestorm; Ramirez DePietro, Pillager; Tor Wauki the Younger

### PR DESCRIPTION
**Pull request by Suthro**
DMU + DMC: Add scripts for one DMU card and two DMC cards:

- Temporal Firestorm
- Ramirez DePietro, Pillager
- Tor Wauki the Younger

***** Update 7/27/22 1:55PM CST

DMC: Add script for Jasmine Boreal of the Seven

- Jasmine Boreal of the Seven

### CODE REVIEW

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/jasmine_boreal_of_the_seven.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Jasmine Boreal of the Seven
+ManaCost:1 G W
+Types:Legendary Creature Human Druid
+PT:2/4
+A:AB$ Mana | Cost$ T | Produced$ G W | RestrictValid$ Spell.Creature+NoAbilities | SpellDescription$ Add {G}{W}. Spend this mana only to cast creature spells with no abilities.
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.NoAbilities+YouCtrl | ValidBlocker$ Creature.!NoAbilities | Description$ Creatures you control with no abilities can’t be blocked by creatures with abilities.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/jasmine_boreal_of_the_seven.txt:6]: `’` -> `'` also in Oracle

File: forge-gui/res/cardsfolder/upcoming/ramirez_depietro_pillager.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigDig:DB$ Dig | Defined$ TriggeredTarget | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | RememberObjects$ Remembered | Duration$ Permanent | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered+nonLand | AffectedZone$ Exile | Description$ You may cast that card for as long as it remains exiled.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHints:Type$Pirate
+Oracle:When Ramirez DePietro, Pillager enters the battlefield, you lose 2 life and create two Treasure tokens.\nWhenever one or more Pirates you control deal combat damage to a player, exile the top card of that player’s library. You may cast that card for as long as it remains exiled.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ramirez_depietro_pillager.txt:14]: `’` -> `'`

File: forge-gui/res/cardsfolder/upcoming/ramirez_depietro_pillager.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigLoseLife:DB$ LoseLife | LifeAmount$ 2 | SubAbility$ TrigToken
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ c_a_treasure_sac
+T:Mode$ DamageDoneOnce | ValidSource$ Pirate.YouCtrl | TriggerZones$ Battlefield | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDig | TriggerDescription$ Whenever one or more Pirates you control deal combat damage to a player, exile the top card of that player's library. You may cast that card for as long as it remains exiled.
+SVar:TrigDig:DB$ Dig | Defined$ TriggeredTarget | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | RememberObjects$ Remembered | Duration$ Permanent | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered+nonLand | AffectedZone$ Exile | Description$ You may cast that card for as long as it remains exiled.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHints:Type$Pirate
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ramirez_depietro_pillager.txt:13]: also new line
`DeckHas:Ability$Token|Sacrifice & Type$Artifact|Treasure`

File: forge-gui/res/cardsfolder/upcoming/temporal_firestorm.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Temporal Firestorm
+ManaCost:3 R R
+Types:Sorcery
+K:Kicker:1 W:1 U
+A:SP$ ChooseCard | Choices$ Creature.YouCtrl,Planeswalker.YouCtrl | Amount$ X | MinAmount$ 0 | ChoiceTitle$ Choose up to X creature and/or planeswalkers you control | SubAbility$ DBPhase | StackDescription$ SpellDescription | SpellDescription$ Choose up to X creatures and/or planeswalkers you control, where X is the number of times this spell was kicked. Those permanents phase out. (While they're phased out, they're treated as though they don't exist. Each one phases in before its controller untaps during their next untap step.) CARDNAME deals 5 damage to each creature and each planeswalker.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/temporal_firestorm.txt:5]: remove reminder text from SpellDescription

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/temporal_firestorm.txt:5]: `X creature` -> `X creatures` in ChoiceTitle

**Review by Northmoc — COMMENTED** [review]
Thank you!

**Comment by Suthro** [comment]
@Northmoc Made the requested changes. Thanks for the peer review 👍

**Review by tool4ever — APPROVED** [review]

**Pull request merged**